docs/segment.md

```diff
@@ -2,98 +2,188 @@
+
+![image: Ontology tab](img/segment/ontology_tab.png "Illustration of the ontology tab.")
+
+eScriptorium uses four region types by default:
+
+- Commentary
+
+- Illustration
+
+- Main
+
+- Title
+
+You can use them, or create your own. To deactivate default region types, you can deactivate them by unticking the ones you want to discard, and then by clicking on the update button.
+
+![image: Deactivating default text regions](img/segment/default_text_regions.gif "Modifying the text region ontology by deactivating default region types.")
+
+You can then create your own by writing the text region name you want into the dedicated field and then the + button. Don't forget to click on the update button to save your modifications!
+
+![image: Creating a custom ontology](img/segment/create_text_regions_ontology.gif "Adding region types to the text region ontology.")
+
+Then, once you are back to the segmentation panel, you can add a label to the bounding boxes you drew earlier by clicking on them, still in region mode, and then by clicking on the 'Set the type on all selected lines/regions' button, or by pressing ++t++. A dropdown menu should appear with the custom labels you created.
+
+![image: Labeling a text region ](img/segment/labeling_text_regions.gif "We click on a text region and label it with a label from the text region ontology.")
 
-![image: Screenshot of the 5 activated panels (Segmentation panel framed in red) ](img/segment/segment_panel.png "The 3d panel is used to manually edit the segmentation")
+!!! tip
+    You can change the color of a text region in eScriptorium in the **images** tab by clicking on **editor settings**. 
 
-## Overview of the segmentation panel
+!!! note
+    It is possible to create nested text regions, just draw another bounding box on top of the text regions you want to annotate!
 
-<!-- rewrite this section-->
+!!! note
+    It is possible to automatically perform [text regions segmentation](predict.md#predict-the-segmentation) with a machine learning model.
 
-The segment editing panel allows you to perform several essential operations:
+## Text lines segmentation
 
-- Drawing baselines corresponding to the locations of the text on the image in two ways:
-    - Point by point plot;
-    - Free route (not recommended).
-- Adding points on a line (select it then double-click at the point’s location);
-- Moving one or more point(s) by a simple select-drag;
-- Deleting one or more point(s) on a line;
-- Cutting one or more lines using the scissors tool;
-- Joining one or more lines;
-- Configuring the reading direction of a line;
-- Activating the calculation of the polygons associated with each line (this task is automatic and managed asynchronously without any action on the part of the user after the first use).
+Text lines segmentation is a mandatory process in eScriptorium in order to transcribe a text. In eScriptorium, text lines are composed with two elements:
```

Comment:
I am not sure "composed with" is right, it feels weird to me when I read it --> "composed of" or "made of" ?